### ISSUE TITLE
xorg-server 1.19.1 breaks on darwin

### ISSUE THREAD

**Issue by yamad** [NixOS/nixpkgs#23027]
## Issue description

Commit 2dba30af721d452000917a55a21a722331b113df broke xorg-server for OS X Darwin on my machine. It errors out during the patch stage, same as reported in the Travis CI build: https://travis-ci.org/NixOS/nixpkgs/builds/200931915.

The log says:

```
unpacking source archive /nix/store/cdrd23zc0jnnlfi2qczgmjkahjhiphhr-xorg-server-1.19.1.tar.bz2
source root is xorg-server-1.19.1
setting SOURCE_DATE_EPOCH to timestamp 1484169768 of file xorg-server-1.19.1/INSTALL
patching sources
applying patch /nix/store/k11g8drjmwkaax67pv6zbgxgw0jg941d-0002-sdksyms.sh-Use-CPPFLAGS-not-CFLAGS.patch
patching file hw/xfree86/Makefile.am
Hunk #1 FAILED at 48.
Hunk #2 succeeded at 137 (offset 2 lines).
1 out of 2 hunks FAILED -- saving rejects to file hw/xfree86/Makefile.am.rej
```

### Steps to reproduce

1) Sync nixpkgs past commit 2dba30
2) nix-env -i xorg-server

## Technical details

* System: Darwin, OS X 10.11.6
* Nix version: 1.11.6
* Nixpkgs version: 17.03pre101638.8697fb9

**Comment by yamad** [comment]
sorry, actually it looks like this error has been around for awhile. trying to track it down now...

**Comment by vcunat** [comment]
If you/someone actually fix xorg-server 1.19 on Darwin, file a PR and mention me.
